### DISCUSSION TITLE
请问如何最简单的配置特定网域经过代理、其馀直连呢？

### DISCUSSION THREAD

**Discussion by i1itione** [kkkgo/PaoPaoGateWay#113] [问和答]
由于上网环境基本上都是直连即可，因此需求仅为更方便的访问锁当地 IP 的网站，比方说一些限日本 IP 的网站。
目前已经有一组可以使用的日本 IP 的 Socks5 Proxy，本来我是直接在 PC 浏览器（例如 Chrome）搭配扩充功能，
但这个做法的缺点就是家中每一台 PC 上的浏览器都需要单独设置，且不适用于 Android TV 这类设备。

这几天爬了文章，有找到可在 PC 上用虚拟机安装 OpenWRT 运行 Pass Wall 或 PassWall 2 服务来实现分流功能，
这样仅需要将上网设备的网关修改为该虚拟机的内网 IP 即可，让多数情况维持直连，依域名规则才经过 Proxy，
不过现在发现了 PaoPaoGateWay 这个项目，似乎会比专程安装 OpenWRT 仅仅只为了分流更直觉且高效，
想请问 PaoPaoGateWay 确实是可以实现上述使用需求的吗？以及如何最简单的设置来实现依域名分流的效果呢？

**Comment by kkkgo** [comment]
可以使用yaml模式，然后自己写。
参考 [代理](https://wiki.metacubex.one/config/proxies/) 和 [路由规则](https://wiki.metacubex.one/config/rules/)
但不建议这么用，因为这个网关的实际使用场景拓扑是作为全局代理网关，然后配合第三方DNS，依赖FAKEIP的DNS分流。
拓扑原理参考 https://blog.03k.org/post/paopaogateway.html
搭建参考 https://github.com/kkkgo/PaoPaoGateWay/discussions/categories/%E9%85%8D%E7%BD%AE%E5%88%86%E4%BA%AB
